### PR TITLE
Build on Mac OS X ARM64

### PR DESCRIPTION
**Pull request by reznikmm**
As discussed in https://github.com/alire-project/setup-alire/pull/74

### CODE REVIEW

File: .github/workflows/ci-macos-arm64.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    name: CI on macOS
+
+    runs-on: macos-14
+
```
Comment (mosteo) [inline .github/workflows/ci-macos-arm64.yml]: Do you think we could avoid some duplication by having a single `ci-macos.yml` with a few conditional steps?

**Review by mosteo — COMMENTED** [review]
See comment about merging both macos workflows

File: .github/workflows/ci-macos-arm64.yml (hunk 1 above)
Comment (reznikmm) [reply .github/workflows/ci-macos-arm64.yml]: Ok. I've rewritten it in a single file as you advised.

**Comment by reznikmm** [comment]
I've tried a testing release. It creates `alr-2.0.2-bin-aarch64-macos.zip`. Is it appropriated name?

**Comment by reznikmm** [comment]
ping

**Comment by mosteo** [comment]
Yes, I think that's the proper name as we use internally aarch64 over arm64.

Thanks for the rework.

**Review by mosteo — APPROVED** [review]

**Comment by reznikmm** [comment]
You are welcome. If you have suggestion for next step(s) to release `alr` on Mac OS X ARM64, don't hesitate to tell me.

**Comment by mosteo** [comment]
I think with this we are set. I don't want to make a release right now because I have limited time available in case of surprises, but in a couple weeks we can go ahead.

**Pull request merged**